### PR TITLE
[Grouping&Mapping] Improve Search Bar

### PR DESCRIPTION
**Pull request by arinsasmal**

### CODE REVIEW

**Comment by Guillar1** [comment]
lgtm

**Review by Guillar1 — APPROVED** [review]

**Review by Guillar1 — CHANGES_REQUESTED** [review]
One more thing, can you make it so the search clears out when you completely delete the all the text?

File: packages/itwin/grouping-mapping-widget/src/widget/components/SelectProject.tsx [hunk 1]
```diff
@@ -61,8 +63,34 @@ const SelectProject = ({ onSelect, onCancel }: SelectProjectProps) => {
...
+  const clearSearch = useCallback(() => {
+    setSearchInput("");
+    setActiveSearchInput("");
+    setSearched(false);
+  }, []);
+
+  const runLabel = useCallback(() => {
+    if(searched === false) {
+      return (
+        <IconButton onClick={() => startSearch() } styleType='borderless'>
```
Comment (bingjiez) [inline packages/itwin/grouping-mapping-widget/src/widget/components/SelectProject.tsx:78]: Could shorten to onClick={startSearch}

Comment (arinsasmal) [reply packages/itwin/grouping-mapping-widget/src/widget/components/SelectProject.tsx]: Done

**Review by Guillar1 — APPROVED** [review]

**Pull request merged**